package.json

```diff
@@ -1,13 +1,34 @@
 {
   "name": "fluent-javascript",
-  "version": "0.0.1",
   "private": true,
+  "workspaces": [
+    "./fluent-dedent",
+    "./fluent-bundle",
+    "./fluent-sequence",
+    "./fluent-dom",
+    "./fluent-langneg",
+    "./fluent-react",
+    "./fluent-syntax",
+    "./fluent-gecko"
+  ],
+  "scripts": {
+    "predist": "npm run clean",
+    "dist": "npm run build --workspaces",
+    "postdist": "npm run lint && npm run test && npm run docs --workspaces",
+    "clean": "git clean -fdxe node_modules fluent-*/{coverage,dist,esm,index.js} html",
+    "deploy-html": "gh-pages -d html",
+    "lint": "eslint --max-warnings 0 'fluent-*/{src,test}/**/*.{js,ts}'",
```

Comment:
Issue: I get failing lints, it seems that this call should be clean.

```
➤ npm run lint

> lint
> eslint --max-warnings 0 'fluent-*/{src,test}/**/*.{js,ts}'


/Users/greg/dev/fluent.js/fluent-react/src/localization.ts
  30:5   error  Unsafe return of an `any` typed value                  @typescript-eslint/no-unsafe-return
  30:12  error  Unsafe call of an `any` typed value                    @typescript-eslint/no-unsafe-call
  44:11  error  Unsafe assignment of an `any` value                    @typescript-eslint/no-unsafe-assignment
  46:13  error  Unsafe assignment of an `any` value                    @typescript-eslint/no-unsafe-assignment
  46:19  error  Unsafe member access .getMessage on an `any` value     @typescript-eslint/no-unsafe-member-access
  46:19  error  Unsafe call of an `any` typed value                    @typescript-eslint/no-unsafe-call
  47:18  error  Unsafe member access .value on an `any` value          @typescript-eslint/no-unsafe-member-access
  49:13  error  Unsafe assignment of an `any` value                    @typescript-eslint/no-unsafe-assignment
  49:21  error  Unsafe member access .formatPattern on an `any` value  @typescript-eslint/no-unsafe-member-access
  49:21  error  Unsafe call of an `any` typed value                    @typescript-eslint/no-unsafe-call
  49:42  error  Unsafe member access .value on an `any` value          @typescript-eslint/no-unsafe-member-access
  53:9   error  Unsafe return of an `any` typed value                  @typescript-eslint/no-unsafe-return

/Users/greg/dev/fluent.js/fluent-react/src/localized.ts
   75:9   error  Unsafe assignment of an `any` value                    @typescript-eslint/no-unsafe-assignment
  106:9   error  Unsafe assignment of an `any` value                    @typescript-eslint/no-unsafe-assignment
  106:15  error  Unsafe member access .getMessage on an `any` value     @typescript-eslint/no-unsafe-member-access
  106:15  error  Unsafe call of an `any` typed value                    @typescript-eslint/no-unsafe-call
  113:9   error  Unsafe member access .value on an `any` value          @typescript-eslint/no-unsafe-member-access
  115:11  error  Unsafe assignment of an `any` value                    @typescript-eslint/no-unsafe-assignment
  115:19  error  Unsafe member access .formatPattern on an `any` value  @typescript-eslint/no-unsafe-member-access
  115:19  error  Unsafe call of an `any` typed value                    @typescript-eslint/no-unsafe-call
  115:40  error  Unsafe member access .value on an `any` value          @typescript-eslint/no-unsafe-member-access
  130:16  error  Unsafe member access .attributes on an `any` value     @typescript-eslint/no-unsafe-member-access
  134:30  error  Unsafe member access .attributes on an `any` value     @typescript-eslint/no-unsafe-member-access
  135:9   error  Unsafe assignment of an `any` value                    @typescript-eslint/no-unsafe-assignment
  135:32  error  Unsafe member access .formatPattern on an `any` value  @typescript-eslint/no-unsafe-member-access
  135:32  error  Unsafe call of an `any` typed value                    @typescript-eslint/no-unsafe-call
  136:11  error  Unsafe member access .attributes on an `any` value     @typescript-eslint/no-unsafe-member-access
  155:7   error  Unsafe member access .value on an `any` value          @typescript-eslint/no-unsafe-member-access
  160:9   error  Unsafe assignment of an `any` value                    @typescript-eslint/no-unsafe-assignment
  160:24  error  Unsafe member access .formatPattern on an `any` value  @typescript-eslint/no-unsafe-member-access
  160:24  error  Unsafe call of an `any` typed value                    @typescript-eslint/no-unsafe-call
  160:45  error  Unsafe member access .value on an `any` value          @typescript-eslint/no-unsafe-member-access

/Users/greg/dev/fluent.js/fluent-sequence/src/map_async.ts
  26:11  error  Unsafe member access .hasMessage on an `any` value  @typescript-eslint/no-unsafe-member-access
  26:11  error  Unsafe call of an `any` typed value                 @typescript-eslint/no-unsafe-call
  27:9   error  Unsafe return of an `any` typed value               @typescript-eslint/no-unsafe-return
  39:35  error  Unsafe member access .hasMessage on an `any` value  @typescript-eslint/no-unsafe-member-access
  39:35  error  Unsafe call of an `any` typed value                 @typescript-eslint/no-unsafe-call
  40:9   error  Unsafe assignment of an `any` value                 @typescript-eslint/no-unsafe-assignment
  46:9   error  Unsafe return of an `any[]` typed value             @typescript-eslint/no-unsafe-return
  51:3   error  Unsafe return of an `any[]` typed value             @typescript-eslint/no-unsafe-return

/Users/greg/dev/fluent.js/fluent-sequence/src/map_sync.ts
  29:11  error  Unsafe return of an `any` typed value               @typescript-eslint/no-unsafe-return
  41:9   error  Unsafe member access .hasMessage on an `any` value  @typescript-eslint/no-unsafe-member-access
  41:9   error  Unsafe call of an `any` typed value                 @typescript-eslint/no-unsafe-call
  42:7   error  Unsafe return of an `any` typed value               @typescript-eslint/no-unsafe-return

✖ 44 problems (44 errors, 0 warnings)
```

Reply:
Does running the `dist` script before `lint` solve this for you? This is almost certainly caused by inter-package dependencies using compiled rather than source dependencies.

Reply:
`npm run lint` is working now, so it must have relied on `dist`.

Reply:
Suggestion: It seems like the run scripts are failing due to dependency management. Makefiles can run dependent jobs correctly, so it might be worth adding a makefile into the mix to ensure the commands are easy to run. The makefiles could still call npm run scripts.

Reply:
I would actually prefer to solve this by removing the dependencies on the `dist` assets, but that seems like a separate change from starting to use npm workspaces so I've not included it here.